src/data.py

```diff
@@ -293,7 +293,7 @@ def _is_default(prop):
             if ylabel:
                 original_position = new_item.get_yposition()
                 if original_position == 0:
-                    if _is_default("left-label"):
+                    if _is_default("left-label") or not self.items:
```

Comment:
```suggestion
                    if _is_default("left-label") or self.props.empty:
```

Reply:
Of course, agree its better to use the specific property we have for this.